src/NServiceBus.Core/Licensing/AuditInvalidLicenseBehavior.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Diagnostics;
+    using System.Threading.Tasks;
+    using Logging;
+    using Audit;
+    using Pipeline;
+
+    class AuditInvalidLicenseBehavior : Behavior<AuditContext>
+    {
+        public override async Task Invoke(AuditContext context, Func<Task> next)
+        {
+            context.AddAuditData(Headers.HasLicenseExpired, true.ToString().ToLower());
```

Comment:
That would actually be more correct. Otherwise I guess we would need to specify invariant culture.